client/src/components/UserProfilePage.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from "react";
+import UserSaladList from "./UserSaladList"
+
+const UserProfilePage = (props) => {
+    const [user, setUser] = useState([])
```

Comment:
based on the update below, you shouldn't need state for the user because it will be passed via props. However, be mindful in the future about the default data type for state. a single `user` is an object, not an array